modules/genomic_browser/php/filemanager.class.inc

```diff
@@ -0,0 +1,207 @@
+<?php declare(strict_types=1);
+
+namespace LORIS\genomic_browser;
+
+use LORIS\Data\Filters\HasAnyPermissionOrUserSiteMatch;
+use LORIS\genomic_browser\Provisioners\FilesProvisioner;
+use \Psr\Http\Message\ServerRequestInterface;
+use \Psr\Http\Message\ResponseInterface;
+use \LORIS\Middleware\ETagCalculator;
+
+require_once 'uploading/genomicfile.class.inc';
```

Comment:
includes shouldn't be mixed in files with class definitions, this should be autoloaded.